bot/exts/fun/snakes/__init__.py

```diff
@@ -1,11 +1,14 @@
 import logging
 
 from bot.bot import Bot
+from bot.constants import Tokens
 from bot.exts.fun.snakes._snakes_cog import Snakes
 
 log = logging.getLogger(__name__)
 
 
 async def setup(bot: Bot) -> None:
     """Load the Snakes Cog."""
+    if not Tokens.giphy:
+        log.warning("No Youtube token. All youtube related commands in Snakes cog won't work.")
```

Comment:
Looks like the wrong constant? Also capitalise youtube as `YouTube`

Reply:
5bdd195aefc1b3af562e336dd9edeb260002e085

Reply:
> There's currently an error log for `bot.exts.utilities.reddit` when missing the token, could we make that a warning for consistency?

Yessir we can 0e202c04bd6c5976afdcd4d10a62fde386dcf440